src/Views/OperatingSystemView.vala

```diff
@@ -70,9 +70,13 @@ public class About.OperatingSystemView : Gtk.Grid {
 
         // Intentionally not using GLib.OsInfoKey.PRETTY_NAME here because we
         // want more granular control over text formatting
+        var version_name = Environment.get_os_info (GLib.OsInfoKey.VERSION);
+        if (version_name == null) {
+            version_name = "";
+        }
         var pretty_name = "<b>%s</b> %s".printf (
             Environment.get_os_info (GLib.OsInfoKey.NAME),
-            Environment.get_os_info (GLib.OsInfoKey.VERSION)
+            version_name
```

Comment:
```suggestion
            Environment.get_os_info (GLib.OsInfoKey.VERSION) ?? ""
```

You can simplify this by using the [null coalescing operator](https://naaando.gitbooks.io/the-vala-tutorial/content/en/3-basics/operators.html):

> `a ?? b` is equivalent to `a != null ? a : b`. This operator is useful for example to provide a default value in case a reference is _null_:

> ```csharp
> stdout.printf("Hello, %s!\n", name ?? "unknown person");
> ```

Reply:
Thank you very much for you review and advice. I’ve integrated them :)